vendor/github.com/knative/caching/CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contribution guidelines
 
 So you want to hack on Knative `caching`? Yay! Please refer to Knative's overall
-[contribution guidelines](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md)
+[contribution guidelines](https://www.knative.dev/contributing/)
```

Comment:
Things under `vendor/` shouldn't be updated here. We need to update the caching repo: https://github.com/knative/caching and then update `Gopkg.toml` to pick up the change.

Reply:
Removed these files from the PR.